### PR TITLE
Feature/mempool size info chart

### PR DESCRIPTION
**Pull request by Dinisueldanov**
### Summary
- added mempool size info chart.
- added tests.

### Screenshots
main-mempool
mempool

### CODE REVIEW

File: src/features/pastelStatistics/common/constants.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import Difficulty from '../assets/images/difficulty.jpg'
 import PSLPrice from '../assets/images/pslprice.jpg'
 import Hashrate from '../assets/images/hashrate.jpg'
+import Nettotals from '../assets/images/nettotals.jpg'
```
Comment (ngvtuan) [inline src/features/pastelStatistics/common/constants.ts:4]: Why do we need this file?

File: src/features/pastelStatistics/common/constants.ts [hunk 2]
```diff
@@ -19,6 +21,16 @@ export const pastelChartFields = [
     routeName: '/statistics/hashrateovertime',
     backgroundImage: Hashrate,
   },
+  {
+    name: 'Nettotals',
+    routeName: '/statistics/nettotalsovertime',
+    backgroundImage: Nettotals,
+  },
```
Comment (ngvtuan) [inline src/features/pastelStatistics/common/constants.ts:28]: the `Nettotals` chart? Why should we need to include this chart?

File: src/features/pastelStatistics/components/nettotalsovertime/NetTotalsOvertime.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from 'react'
```
Comment (ngvtuan) [inline src/features/pastelStatistics/components/nettotalsovertime/NetTotalsOvertime.tsx:1]: Why do we need this chart?

File: src/features/pastelStatistics/utils/ChartOptions.ts [hunk 4]
```diff
@@ -127,6 +127,173 @@ export function getThemeInitOption(args: TThemeInitOption): EChartsOption {
         easing: 'cubicOut',
       },
     },
+    nettotals: {
```
Comment (ngvtuan) [inline src/features/pastelStatistics/utils/ChartOptions.ts:130]: Why do we need this?

File: src/features/pastelStatistics/utils/ChartOptions.ts [hunk 5]
```diff
@@ -168,6 +335,43 @@ export function getThemeUpdateOption(args: TThemeInitOption): EChartsOption {
   const chartOptions: TChartOption = {
     difficulty: defaultOption,
     hashrate: defaultOption,
+    nettotals: {
```
Comment (ngvtuan) [inline src/features/pastelStatistics/utils/ChartOptions.ts:338]: Same here

File: src/legacy/Routes.tsx [hunk 6]
```diff
@@ -51,6 +51,8 @@ import {
   PriceOvertime,
   PastelStatistics,
   HashrateOvertime,
+  NetTotalsOvertime,
```
Comment (ngvtuan) [inline src/legacy/Routes.tsx:54]: Same here

File: src/legacy/constants/routes.json [hunk 7]
```diff
@@ -13,6 +13,8 @@
   "STATISTICS": "/statistics",
   "PRICEOVERTIME": "/statistics/priceovertime",
   "HASHRATEOVERTIME": "/statistics/hashrateovertime",
+  "MEMPOOLSIZEOVERTIME": "/statistics/mempoolsizeovertime",
+  "NETTOTALSOVERTIME": "/statistics/nettotalsovertime",
```
Comment (ngvtuan) [inline src/legacy/constants/routes.json:17]: Same here

File: src/legacy/Routes.tsx [hunk 8]
```diff
@@ -654,6 +656,16 @@ class RouteApp extends React.Component<any, any> {
                 render={() => <HashrateOvertime info={info} />}
               />
 
+              <Route
+                path={routes.NETTOTALSOVERTIME}
+                render={() => <NetTotalsOvertime info={info} />}
+              />
```
Comment (ngvtuan) [inline src/legacy/Routes.tsx:662]: Same here

File: src/features/pastelStatistics/__tests__/pastelLib.test.ts [hunk 9]
```diff
@@ -45,6 +45,15 @@ describe('managePastelDatabase', () => {
       1621518133357,
     ],
   ]
+  const mockNetTotals = [
```
Comment (ngvtuan) [inline src/features/pastelStatistics/__tests__/pastelLib.test.ts:48]: I think this mock belong to the NetTotals chart, right?

**Review by ngvtuan — COMMENTED** [review]
ui issues 1
Why does this chart showing in this PR? 

index ts
We are missing the index.ts file for each component


https://user-images.githubusercontent.com/10944387/120789509-e6371f00-c55b-11eb-8b60-ae83d4c0145c.mov
- Can we fix the dragging issue? 
- Can we change the chart name to `Mempool Size`? 

Please remove all code that isn't needed for this PR.

File: src/features/pastelStatistics/__tests__/pastelLib.test.ts (hunk 9 above)
Comment (Dinisueldanov) [reply src/features/pastelStatistics/__tests__/pastelLib.test.ts:48]: This is the nettotals table's mock data.

Comment (ngvtuan) [reply src/features/pastelStatistics/__tests__/pastelLib.test.ts:48]: Ok So please remove it because it is not related to this PR.

File: src/legacy/Routes.tsx (hunk 8 above)
Comment (ngvtuan) [reply src/legacy/Routes.tsx:662]: Please remove un-needed code.

File: src/features/pastelStatistics/utils/PastelStatisticsLib.ts [hunk 10]
```diff
@@ -119,3 +119,52 @@ export function transformHashrateInfo(
 
   return { dataX, dataY }
 }
+
+export function transformNetTotals(
```
Comment (ngvtuan) [inline src/features/pastelStatistics/utils/PastelStatisticsLib.ts:123]: Please remove un-needed code.

File: src/features/pastelStatistics/components/networktotalsovertime/NetworkTotalsOvertime.tsx [hunk 11]
```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from 'react'
```
Comment (ngvtuan) [inline src/features/pastelStatistics/components/networktotalsovertime/NetworkTotalsOvertime.tsx:1]: This file doesn't belong to this PR. Please remove

**Review by ngvtuan — COMMENTED** [review]
Please remove un-needed code/files

**Review by ArthurVogelsang — DISMISSED** [review]

**Review by ArthurVogelsang — APPROVED** [review]

**Review by ngvtuan — APPROVED** [review]
